buf.gen.ts.yaml

```diff
@@ -1,21 +1,24 @@
 version: v2
+clean: true
 plugins:
-  - local: protoc-gen-es
+  - local: ./ts/node_modules/.bin/protoc-gen-es
     strategy: all
-    out: ./ts/src/generated
+    out: ./ts/src/generated/protos
     opt:
       - target=ts
       - json_types=true
-  - local: sdk-object
-    path: ./ts/bin/protoc-sdk-object.ts
-    out: ./ts/src/gen
+  - local: ./ts/bin/protoc-sdk-object.ts
```

Comment:
add `ts/bin` to path in `.envrc`
`PATH_add ${ROOT_DIR}/ts/bin`

then `local: protoc-sdk-object.ts`

Reply:
actually, there is `scripts` dir which is better location for this file

Reply:
The same with this file. I’d not call it a script because it’s a special plugin script. 

I put it in bin in similar way as node_modules binaries live in node_modules/.bin

adding it to PATH also doesn’t make sense to me since it’s not common to use it outside buf configs